libraries/Microsoft.Bot.Connector/Authentication/GovernmentAuthenticationConstants.cs

```diff
@@ -18,11 +18,21 @@ public static class GovernmentAuthenticationConstants
         /// </summary>
         public const string ToChannelFromBotLoginUrl = "https://login.microsoftonline.us/cab8a31a-1906-4287-a0d8-4eef66b95f6e";
 
+        /// <summary>
+        /// TO GOVERNMENT CHANNEL FROM BOT: Login URL V2.
+        /// </summary>
+        public const string ToChannelFromBotLoginUrlV2 = "https://login.microsoftonline.us/botframework.onmicrosoft.us";
+
         /// <summary>
         /// TO GOVERNMENT CHANNEL FROM BOT: OAuth scope to request.
         /// </summary>
         public const string ToChannelFromBotOAuthScope = "https://api.botframework.us";
 
+        /// <summary>
+        /// TO GOVERNMENT CHANNEL FROM BOT: OAuth scope to request V2.
+        /// </summary>
+        public const string ToChannelFromBotOAuthScopeV2 = "https://api.botframework.onmicrosoft.us";
```

Comment:
This line is really the only line I expected to see added.

Reply:
It's the hardcoded tenant above (cab8a31a-1906-4287-a0d8-4eef66b95f6e - USME) that needed to be updated to a different tenant.  And then

1. the scope (api.botframework.onmicrosoft.us) is trivially name-spaced within the resource app identifier (api.botframework.onmicrosoft.us), and
2. the app identifier (api.botframework.onmicrosoft.us) is name-spaced within the tenant domain (botframework.onmicrosoft.us, which we may modify to a custom domain botframework.azure.us to match our DNS name).